### PR TITLE
Updated links in Summary and Consensus definition

### PR DESCRIPTION
**Pull request by FieldmanJR1**

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Byzantine fault tolerance (BFT) is a characteristic of a distributed system that
 
 ## Consensus
 
-Consensus is the process of a group of entities to agree on a particular data value (such as the ordering and makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus. GGX uses hybrid consensus - [BABE](#babe) for block production and [GRANDPA](#grandpa) as a finality gadget.
+Consensus is the process of a group of entities to agree on a particular data value (such as the ordering and makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus. GGX uses hybrid consensus - [BABE](babe) for block production and [GRANDPA](grandpa) as a finality gadget.
```
Comment (xcthulhu) [inline docs/glossary.md:25]: I don't think we should be getting rid of those hashes
```suggestion
Consensus is the process of a group of entities to agree on a particular data value (such as the ordering and makeup of blocks on a blockchain). There are a variety of algorithms used for determining consensus. GGX uses hybrid consensus - [BABE](#babe) for block production and [GRANDPA](#grandpa) as a finality gadget.
```

Comment (xcthulhu) [reply docs/glossary.md:25]: Also, we don't use BABE for block production we use AURA

File: SUMMARY.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-- Wallet
-  - Metamask
-  - Polkadot.js
+- [Wallet](docs/wallet.md)
+  - [Metamask](docs/metamask.md)
+  - [Polkadot.js](docs/polkadotjs.md)
 - Deploying a Smart Contract
-  - Rust-based WASM contracts
+  - [Rust-based WASM contracts](docs/how_to_deploy_rust.md)
   - Javascript-based WASM contracts
```
Comment (xcthulhu) [inline SUMMARY.md:12]:
```suggestion
```

File: SUMMARY.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-  - Solidity-based EVM contracts
+  - [Solidity-based EVM contracts](docs/how_to_deploy_solidity.md)
 - Running a Validator
   - Generate Validator Account
   - Deploy Validator Node
```
Comment (xcthulhu) [inline SUMMARY.md:16]:
```suggestion
```

File: SUMMARY.md (hunk 2 above)
Comment (FieldmanJR1) [reply SUMMARY.md:12]: Ok - deleting

**Review by crawter — APPROVED** [review]

**Pull request merged**